manifests/SanfordLP/DYMOLabel/8.7.3.46663.yaml

```
@@ -0,0 +1,18 @@
+Id: SanfordLP.DYMOLabel
+Version: 8.7.3.46663
+Name: DYMO Label
+Publisher: SanfordLP
+License: Copyright © 2016 Sanford, L.P.
+AppMoniker: DYMOLabel
+Tags: Dymo, LabelWriter, LabelWriter 400, LabelWriter 400 Turbo, LabelWriter Twin Turbo, LabelWriter Duo, LabelManager Wireless PnP, LabelManager 450, LabelManager 450D, LabelManager PCII, LabelPoint 350
+Description: DYMO Label Installer
+Homepage: https://www.dymo.com/en-US/dymo-label-software-v873-windows
+Installers:
+- Arch: x86
+  Url: http://download.dymo.com/dymo/Software/Win/DLS8Setup.8.7.3.exe
+  Sha256: D7B55A5E1431D5B44241BEE91D1841A5CD1990480DFCB73C9F80F57DC6B9C803
+  InstallerType: exe
+  Scope: machine
+ Switches:
```

```suggestion
  Switches:
```

`Scope` and `Switches` must be aligned.

---

Thank you.